src/network/SessionService.cpp

```
@@ -0,0 +1,298 @@
+        emit operationFailed(m_errorString);
+        return;
+    }
+
+    // First refresh the session list
+    fetchActiveSessions();
+
+    // Wait for sessions to load, then revoke all except current
+    connect(this, &SessionService::sessionsLoaded, this, [this]() {
+        int revokedCount = 0;
+        for (const QVariant &var : m_sessions) {
+            QVariantMap session = var.toMap();
```

Cppcheck (2.19.0)

[error] There is an unknown macro here somewhere. Configuration is required. If slots is a macro then please configure it.

(unknownMacro)